pulpcore/pulpcore/app/tasks/repository.py

```diff
@@ -109,3 +109,40 @@ def delete_version(pk):
 
         # With no more relationships remaining, delete the version.
         version.delete()
+
+
+@shared_task(base=UserFacingTask)
+def create_version(repository_pk, add_content_units, remove_content_units):
```

Comment:
could we call this add_or_remove? This name could confuse plugin writers because this is not involved in other tasks that create a version